staging/src/k8s.io/apiserver/pkg/storage/etcd3/store.go

```diff
@@ -738,28 +747,56 @@ func (s *store) getStateFromObject(obj runtime.Object) (*objState, error) {
 	return state, nil
 }
 
-func (s *store) updateState(st *objState, userUpdate storage.UpdateFunc) (runtime.Object, uint64, error) {
-	ret, ttlPtr, err := userUpdate(st.obj, *st.meta)
+var errStaleTTL = errors.New("updateState needs current objState for TTL calculation")
```

Comment:
Move this to `etcd3/errors.go` and make it a const?

Is 'stale' the right word here? That imply something has exceeded its shelf life. But the comment suggests the objState is missing needed data for a TTL calculation.

Reply:
`errIncompleteTTL`?